google-cloud-spanner-hibernate-testing/src/test/java/org/hibernate/userguide/hql/HQLTest.java

```diff
@@ -1469,24 +1432,6 @@ public void test_hql_collection_expressions_example_4() {
 		});
 	}
 
-	@Test
```

Comment:
Instead of deleting tests that don't work, I would just put `@Ignore` on them with a short description of why.

Reply:
Done.